Workbooks/Azure Backup/Backup Explorer/settings.json

```diff
@@ -3,5 +3,5 @@
     "name":"Backup Explorer",
     "author": "Microsoft",
     "isPreview": true,
-    "galleries": [{ "type": "workbook", "resourceType": "Azure Monitor", "order": 100 }]
+    "galleries": [{ "type": "backup-insights", "resourceType": "Azure Monitor", "order": 100 }]
```

Comment:
i'd keep both galleries for now while your stuff is getting deployed.  once the new toc item appears, then you could remove the "old" gallery